specification/postgresql/Microsoft.OrionDB.Management/examples/2025-08-01-preview/Clusters_Update.json

```diff
@@ -0,0 +1,114 @@
+{
+  "operationId": "Clusters_Update",
+  "title": "Update an OrionDB cluster",
+  "parameters": {
+    "subscriptionId": "ffffffff-ffff-ffff-ffff-ffffffffffff",
+    "resourceGroupName": "TestGroup",
+    "orionDbClusterName": "testcluster1",
+    "api-version": "2025-08-01-preview",
+    "parameters": {
+      "tags": {
+        "Environment": "Production",
+        "Team": "Database",
+        "Owner": "DataTeam"
+      },
+      "properties": {
+        "administratorLoginPassword": "newPassword123!",
+        "storage": {
+          "storageSizeGB": 1024,
+          "autoGrow": "Enabled",
+          "tier": "P20"
+        },
+        "backup": {
+          "retentionDays": 14,
+          "geoRedundantBackup": "Enabled"
+        },
+        "maintenanceWindow": {
+          "customWindow": "Enabled",
+          "dayOfWeek": 6,
+          "startHour": 3,
+          "startMinute": 30
+        }
+      },
+      "sku": {
```

Comment:
SKU initially will be just #Cores  and ultimately will have a Database generation. We wont expose GP and memory optimized etc and not individual VM SKUs

Reply:
ok I will confirm this with Yogesh and update it accordingly